### ISSUE TITLE
incorrect example

### ISSUE THREAD

**Issue by euri10** [GoogleCloudPlatform/python-docs-samples#1183]
## In which file did you encounter the issue?

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/pubsub/cloud-client/subscriber.py#L33-L35

### Did you change the file? If so, how?

list_subscriptions takes the project_path as parameter, not the topic_path

correct method would be
```
def list_subscriptions(project):
    """Lists all subscriptions for a given topic."""
    subscriber = pubsub_v1.SubscriberClient()
    project_path = subscriber.project_path(project)

    for subscription in subscriber.list_subscriptions(project_path):
        print(subscription.name)
```

**Comment by michaelawyu** [comment]
@jonparrott I think the problem would be solved if pull request #1169 is merged :)

**Comment by jonparrott** [comment]
donion rings.
